README.md

```diff
@@ -63,7 +63,43 @@ datetime | consumption
 
 To work on the challenge, please fork this repository or download it. After you have finished, you can send us a link to the fork or simply zip the repository and email it to us.
 
-### Questions?
+
+## The Frontend Challenge
+#### Setup
+The frontend challenge is located in the project folder `frontend`. To be able to use APIs you have to setup the dev environment:
+* Create the dev environment with `virualenv venv`.
+* Activate environment: `source venv/bin/activate`.
+* Install requirements: `pip install -r requirements.txt`.
+
+After setup, please move to frontend folder (`cd frontend`) and run web server with `python manage.py runserver`. 
+
+If you visit the browser at `http://localhost:8000` you will have to see a page with title **Frontend challenge.**
+
+For the purpose of challenge, we prepared several apis you should use:
+* `api/consumers` will return the list of all consumers;
+* `api/consumers/<low|high|extra_high>` will return consumers of certain type (low voltage, high voltage, extra high voltage);
+* `api/consumer/<consumer_id>` will return a particular consumer (GET) or delete (DELETE) a consumer;
+* a POST request with params `name` and `consumer_type` to  `api/consumer` will create a new consumer;
+* `api/monthly_statistics/<consumer_id>` will return a monthly statistics of a particular consumer with consumption, total_bill and total_cost data points. If you specify year param as: `api/monthly_statistics/<consumer_id>?year=2017` the api returns filtered results by year. The data contains 2016 and 2017. 
+
```

Comment:
@kristjanstrojan How about adding a random load-time delay (0 ~ 5 seconds) to each API response? This will force the developer to show us how they handle loading animations, which is very common for us at SMAP.

Reply:
Added.

Reply:
great idea